src/Uno.UI/UI/Input/PointerPoint.cs

```diff
@@ -37,6 +38,13 @@ internal PointerPoint(
 			Properties = properties;
 		}
 
+		/// <summary>
+		/// Retrieves the relative timestamp in microseconds.
+		/// </summary>
+		/// <returns>Timestamp in microseconds.</returns>
+		internal static ulong GetRelativeTimestamp() =>
```

Comment:
Relative to what ?

Reply:
BTW I realized that I didn't saw any usage of this. If this still use-full?

Reply:
Yes, no longer needed, will remove